content/yaml/yaml-skip-build-and-build-steps.md

```diff
@@ -0,0 +1,136 @@
+---
+
+## Skip building a specific commit
+
+Include `[skip ci]` or `[ci skip]` in your commit message, if you do not wish Codemagic to build a particular commit.
+
+## Using `when` to skip builds
+
+Add `when` key to a workflow root to either skip it or run it, depending on the specified `changeset` and `condition`.
+
+For example, a workflow or script step will run if recent changes include changes to Markdown files (`.md`) *and* the current build is PR update webhook and the PR not draft PR.
+
+```yaml
+when:
+  changeset:
+    includes:
+      - '**/*.md'
+  condition: event.pull_request.draft == true
+```
+
+
+### Using `changeset` inside `when`
+
+You can avoid unnecessary builds when functional components of your repository were not modified. Use conditional workflow triggering to skip building the workflow if the watched files were not updated since the last successful build.
+
+You should specify the files to watch in `changeset` by using the `includes` and `excludes` keys.
+
+```yaml
+workflows:
+  build-app:
+    name: Build App
+    triggering:
+      events:
+        - push
+    when:
+      changeset:
+        includes:
+          - '.'
+        excludes:
+          - '**/*.md'
+```
+
+In this case, the build would be skipped if there were changes only to Markdown files `.md`.
+
+Note that `codemagic.yaml` is always included in the changeset by default.
+
+Both keys `includes` and `excludes` in `changeset` are *optional*. If the `includes` key is not specified, its value will default to `'.'`. The `excludes` key defaults to no exclusions.
+
+If you use a monorepo, each workflow could be responsible for building a part of your application. Use conditional workflow triggering and specify the path to the application in the changeset as in the example below.
+
+```yaml
+workflows:
+  build-android:
+    name: Build Android
+    triggering:
+      events:
+        - push
+    when:
+      changeset:
+        includes:
+          - 'android/'
+```
+
+As a result, commits with changes outside of the android folder will not trigger a build.
+
+
+### Using `condition` inside `when`
+
+`condition` will be evaluated during the build. Build will be skipped if condition evaluates to `false`.
+
+Current environment is accessible under `env` variable.
+
+Examples
+
+```python
+env.ENV == 'debug'  # build will run if ENV is equal to `debug`
+env.PLATFORM == 'windows'  # build will run if PLATFORM is equal to `windows`
+env.FCI_BRANCH != 'master' # build will be skipped if the current branch is not master
+```
+
+Webhook payload is accessible under `event` variable. Note that `event` is not available if build started manually or by a schedule.
+
+Be sure to check the webhook event body in application settings under Webhooks tab.
+
+For the purpose of giving an example, let's assume that webhook body is equal to the following structure
+
+```json
+{
+  "pull_request": {"draft": true},
+  "repository": {"open_issues_count": 3},
+
+  ...
+}
+```
+
```

Comment:
Add sth like this:
You can then specify the following conditions based on the information in the webhook payload.